full-service/src/service/transaction_builder.rs

```diff
@@ -1093,11 +1135,9 @@ mod tests {
             .add_recipient(recipient.clone(), 7_000_000 * MOB, Mob::ID)
             .unwrap();
 
-        match builder.select_txos(&wallet_db.get_conn().unwrap(), None) {
-            Ok(_) => panic!("Should not be able to select txos with > u64::MAX output value"),
-            Err(WalletTransactionBuilderError::OutboundValueTooLarge) => {}
-            Err(e) => panic!("Unexpected error {:?}", e),
-        }
+        builder
+            .select_txos(&wallet_db.get_conn().unwrap(), None)
+            .unwrap();
```

Comment:
Would be nice to verify that the transaction that gets built actually contains the expected output values and change.

Reply:
Why was this marked as resolved?

Reply:
I've expanded the test to include the suggestion

Reply:
Just kidding, this test can be removed (and was now) because it was covered in a different (new) test